src/style.css

```diff
@@ -1,5 +1,14 @@
 /* global styles
 css modules not applied */
+html,
+body {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    padding: 0;
+    overflow: hidden;
```

Comment:
why did you make this change?

Reply:
This makes it so the main page can't scroll

Reply:
dang, ok. i was trying to solve a few issues: 
multiple overflow scroll bars appearing, unsightly and hard to navigate
buttons on the bottom of the page being cut off
a white border appearing at bottom and right of overlay

Reply:
interesting. I haven't seem the multiple overflow scroll bars, but that would be concerning, I'm curious to see what it looks like on your computer